src/lambdas/common-platform-report/src/sendEmail.ts

```diff
@@ -0,0 +1,28 @@
+import { PromiseResult } from "@bichard/types"
+import config from "./config"
+import { TimeRange } from "./generateDates"
+import generateReport from "./generateReport"
+import { ReportRecord } from "./getReportData"
+import Email from "./types/Email"
+import Emailer from "./types/Emailer"
+
+export default async (emailer: Emailer, range: TimeRange, records: ReportRecord[]): PromiseResult<boolean> => {
+  const options: Email = {
+    to: config.toAddresses,
+    from: config.fromAddress,
+    subject: "Bichard 7 Common Platform Error Report",
+    text: `Between the dates ${range.start} and ${range.end} there were ${records.length} errors reported in Bichard`
+  }
+  if (records.length > 0) {
+    const content = generateReport(records)
+    options.attachments = [
+      {
+        filename: `bichard7-error-report-${range.start.toISOString()}.csv`,
+        content
+      }
+    ]
+  }
+  await emailer.sendMail(options)
```

Comment:
minor: can we figure out or output to console if we have a failure when sending the mail? or does it not return a status?

Reply:
Agreed - this is kind of an issue with the code I copied across from User Service as it returns `any`. Will add a ticket to fix there